unity-client/Assets/Scripts/MainScripts/DCL/Controllers/StickersController/StickersController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+
+public class StickersController : MonoBehaviour
+{
+    private StickersFactory stickersFactory;
+
+    private void Awake() { stickersFactory = Resources.Load<StickersFactory>("StickersFactory"); }
+
+    public void PlayEmote(string id)
+    {
+        if (stickersFactory == null || !stickersFactory.TryGet(id, out GameObject prefab))
+            return;
+
+        GameObject emoteGameObject = Instantiate(prefab);
```

Comment:
When/Where is this GO destroyed ?

Reply:
It's a condition of the particle itself to be destroyed automatically. You can do it through the `ParticleSystem` itself or with an util script https://github.com/decentraland/explorer/blob/master/unity-client/Assets/Scripts/MainScripts/DCL/Helpers/Utils/DestroyParticlesOnFinish.cs

Reply:
Awsome, I didn't remember that, thanks!